src/laser/MultiLaser.cpp

```diff
@@ -1063,9 +1258,9 @@ MultiLaser::InitLaserSlice (const amrex::Geometry& geom, const int islice, const
                 bx,
                 [=] AMREX_GPU_DEVICE(int i, int j, int k)
                 {
-                    const amrex::Real z = plo[2] + (islice+0.5_rt)*dx_arr[2] - z0;
```

Comment:
I changed how x, y, z is calculated because the old way was wrong if the laser zeta axis was smaller compared to the full domain.